tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -38,8 +38,8 @@ DEEPBUFFER_D0I3_COMPATIBLE=true"
 SSP1_CORE_ID=1,SSP2_CORE_ID=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-nocodec.bin"
 
 # SSP topology for LNL FPGA with lower DMIC IO clock of 19.2MHz
-"cavs-nocodec\;sof-lnl-nocodec-fpga\;PLATFORM=lnl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,\
-PDM1_MIC_B_ENABLE=1,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-lnl-nocodec-fpga.bin,\
+"cavs-nocodec\;sof-lnl-nocodec-fpga\;PLATFORM=lnl,NUM_DMICS=2,PDM1_MIC_A_ENABLE=1,\
+PDM1_MIC_B_ENABLE=1,PDM0_MIC_A_ENABLE=0,PDM0_MIC_B_ENABLE=0,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-lnl-nocodec-fpga.bin,\
```

Comment:
@ranj063 we should have three topologies
a) 4ch
b) 2ch PDM0
c) 2ch PDM1

that way all the bases are covered - which I never realized is a baseball analogy... Maybe it works with cricket as well.

Reply:
Fixed now